app/controllers/forecasting/distributions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Forecasting::DistributionsController < ApplicationController
+  def index
+    @series     = series
+    @categories = categories
+  end
+
+  private
+
+  def series
+    items = []
+
+    Item.all.sort.each do |item|
+      next if item.line_items.empty?
```

Comment:
I'm not completely clear on the intent here, so to clarify the difference (check the "Application Overview" doc if you have time -- it explains the bigger concepts and vocabulary used):  `line_item` is used anytime inventory is being moved around within the application (sorta like a "ledger line item" or a "manifest line"), and `inventory_item` is used to track inventory at-rest (currently, only in storage locations).

So what this guard clause means, contextually, is "Skip this item if it has never been used for anything ever". The type of situation that would be is if the diaper bank has an item type they are able to redistribute ("6T Diapers", for example), but they have never received any donations or purchased any before, nor have they distributed any. Basically, that item type has never passed through this diaper bank.

We've used this kind of guard clause elsewhere, so if that's the intent, you're all good!

Reply:
I understand now thanks for this comment. I'm using `line_items` and get the `itemizable_type` with `Distribution`.